src/module/helperCode/Equipment Checker.py

```
@@ -88,8 +89,9 @@ def magicPrice (price,magicBonus):
 
                 if data["type"] == "weapon" and data['folder'] != "6Glc0xFoDvY8BJYX" :
                     versatile = False
-
+                    data["system"]["unidentified"]["description"] = "<p><em>This weapon is unusually keen and easy to handle. It is most likely a magic weapon, though you can’t identify the specific enchantment.</em></p>"
```

Ruff (0.7.0)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)